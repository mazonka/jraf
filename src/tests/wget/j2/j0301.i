command=jw md 0 /aaa/ : put 0 /aaa/bbb 0 10 MTIzNDU2Nzg5MA== : put 0 /aaa/bbb 10 10 MTIzNDU2Nzg5MA== : get 0 /aaa : read 0 /aaa : read 0 /aaa/bbb : save 0 /aaa/bbb 10 MTIzNDU2Nzg5MA== : read 0 /aaa/bbb