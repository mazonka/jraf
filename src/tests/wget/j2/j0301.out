OK /aaa : OK 10 : OK 20 : OK 3 -1 : OK 3 -1 1 2 20 bbb : OK 2 20 MTIzNDU2Nzg5MDEyMzQ1Njc4OTA= : OK 10 : OK 3 10 MTIzNDU2Nzg5MA==